topostats/run_topostats.py

```diff
@@ -388,22 +400,8 @@ def main(args=None):
         config["plotting"]["plot_dict"], schema=PLOTTING_SCHEMA, config_type="YAML plotting configuration file"
     )
 
-    # FIXME : Make this a function and from topostats.utils import update_plot_dict and write tests
```

Comment:
Thank you for fixing my long-standing `FIXME`, very much appreciated :smiley_cat: